src/czml3/types.py

```diff
@@ -32,6 +32,21 @@ def format_datetime_like(dt_object):
     return result
 
 
+class FontValue(BaseCZMLObject):
+    """A font, specified using the same syntax as the CSS "font" property."""
+
+    def __init__(self, *, font=None):
+        self._font = font
+
+    @property
+    def font(self):
+        """The font to use for the label."""
+        return self._font
+
+    def to_json(self):
+        return list(self._font)
```

Comment:
Is this a list or a string?

Reply:
My mistake! I also misunderstood how ``Font`` objects are parsed (I thought they're separate objects, while in reality they are simply strings). However, everything should work as expected now (also hit 100% coverage!)

Reply:
Awesome! :grin: